migrations/2023-02-20-101809_update_merchant_connector_account/down.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE merchant_connector_account
```

Comment:
It would have been better if the migration directory names provided a hint on the changes being made. Same for the other migration directory name as well.

Reply:
sure will try to name if possible as a hint of change in the future